src/libraries/System.Runtime.Extensions/tests/System/MathF.cs

```diff
@@ -179,6 +179,24 @@ private static string ToStringPadded(float value)
             }
         }
 
+        [Fact]
+        public static void E()
+        {
+            Assert.Equal(unchecked((int)0x402DF854), BitConverter.SingleToInt32Bits(MathF.E));
+        }
+
+        [Fact]
+        public static void Pi()
+        {
+            Assert.Equal(unchecked((int)0x40490FDB), BitConverter.SingleToInt32Bits(MathF.PI));
+        }
+
+        [Fact]
+        public static void Tau()
+        {
+            Assert.Equal(unchecked((int)0x401921FB60000000), BitConverter.SingleToInt32Bits(MathF.Tau));
```

Comment:
Copy paste error here. I also wouldn't expect you to need the `unchecked((int)value)` (same for `long` above) since the value is already signed and within the right range